lib/src/gainmapmath.cpp

```diff
@@ -463,7 +463,6 @@ ColorTransformFn getInverseOetf(uhdr_color_transfer_t transfer) {
 float getMaxDisplayMasteringLuminance(uhdr_color_transfer_t transfer) {
   switch (transfer) {
     case UHDR_CT_LINEAR:
-      // TODO: configure maxCLL correctly for linear tf
```

Comment:
I think we need to keep the "TODO"

Reply:
as maxCLL is made configurable via ultrahdr api, i have remove this comment dichen